supporting_content/dimensions/availabilityzone.md

```diff
@@ -0,0 +1,38 @@
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                 | Column                    |
+|----------|--------------------------|---------------------------|
+| AWS      | CUR                      | lineItem/AvailabilityZone |
+| GCP      | Big Query Billing Export | location.zone             |
+| Azure    | Cost details             | Not available             |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider | Data set                 | Example Value                                  |
+|----------|--------------------------|------------------------------------------------|
+| AWS      | CUR                      | us-east-1a, me-south-1, ap-southeast-4	   |
+| GCP      | Big Query Billing Export | us-central1-a, europe-west6-b, asia-east2-b  |
+| Azure    | Cost details             | N/A                                            |
+
+## Discussion / Scratch space:
+
+Discussion of zone definition -
+- Original A zone is an identifier assigned to a geographic location by the provider. 
```

Comment:
```suggestion
Discussion of zone definition

- Original A zone is an identifier assigned to a geographic location by the provider.
```

Linter suggestions:

- no trailing punctuation
- list should be surrounded by blank lines
- no trailing space